syncer/util.go

```diff
@@ -118,3 +119,20 @@ func printServerVersion(tctx *tcontext.Context, db *conn.BaseDB, scope string) {
 	dctx := dcontext.NewContext(tctx.Ctx, logger)
 	export.ParseServerInfo(dctx, versionInfo)
 }
+
+const (
+	shadowTableMatch int = iota
+	trashTableMatch
+	allTableMatch
+)
+
+func unmatchedOnlineDDLRules(match int) string {
+	switch match ^ 1 {
```

Comment:
```suggestion
	switch match  {
	case shadowTableMatch
	case trashTableMatch
	case allTableMatch
	}
```
It's a little more readable that way

Reply:
Then `shadowTableMatch` should return `config.TrashTableRules`. I think it's more complicated...